docs/duration.md

```diff
@@ -30,9 +30,9 @@ For more detailed information, see the ISO 8601 standard or the [Wikipedia page]
 | **PT0S**             | Zero |
 | **P0D**              | Zero |
 
-> **NOTE:** According to the ISO 8601 standard, weeks are not allowed to appear together with any other units.
-> As an extension to the standard, Temporal supports combining weeks with other units.
-> If you intend to use a string such as **P3W1D** for interoperability, note that other programs may not accept it.
+> **NOTE:** According to the ISO 8601 standard, weeks are not allowed to appear together with any other units, and durations can only be positive.
+> As extensions to the standard, Temporal supports combining weeks with other units, and a single sign character at the start of the string.
+> If you intend to use a string such as **P3W1D**, **+P1M**, or **-P1M** for interoperability, note that other programs may not accept it.
```

Comment:
I spent a bit of time [looking into this](https://github.com/tc39/proposal-temporal/issues/819#issuecomment-685017374) and I think that it should still be considered an extension because it's in part 2 ("Extensions") of the standard. So the "note that other programs may not accept it" warning still holds. But I'll change the phrasing of this.